internal/slacklog/message.go

```diff
@@ -453,6 +454,26 @@ func (f *MessageFile) DownloadFilename(url, suffix string) string {
 	return filename
 }
 
+// 'text/plain' な添付ファイルをHTMLに埋め込む
+// 存在しない場合、エラーを表示する
+func (f *MessageFile) CodeHTML() (string, error) {
+	suffix := f.DownloadURLsAndSuffixes()[f.URLPrivate]
+	path := "files/" + f.ID + "/" + f.DownloadFilename(f.URLPrivate, suffix)
+	_, err := os.Stat(path)
+	if err != nil {
+		return "ファイルが存在しません", err
+	}
+	src, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "ファイルが読み取れません", err
+	}
```

Comment:
明示的に os.Stat 使うんじゃなくて ioutil.ReadFile の戻り値をチェックすると良いです。あとメッセージは英語のが良いな。

あとファイル1つ読めないだけでgenerator自体が失敗、ってのはちょっとまずいのでこんな感じでエラーを出さないほうが良さげです。

```suggestion
	src, err := ioutil.ReadFile(path)
	if err != nil {
		if os.IsNotExist(err) {
			return fmt.Printf(`<span class="file-error">no files found: %s</span>`, err), nil
		}
		return fmt.Printf(`<span class="file-error">failed to read a file: %s</span>`, err), nil
	}
```